html/semantics/selectors/case-sensitivity-of-selectors/selector-namespace-prefix.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<title>Test whether namespace prefixes are lowercased</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
```

Comment:
Doesn't this test really belong in css/css-namespaces[1], and isn't this already covered by css/css-namespaces/prefix-001.xml ?

[1] https://drafts.csswg.org/css-namespaces-3/#prefixes